config/settings.yml

```diff
@@ -1,6 +1,9 @@
 ---
 :ems:
   :ems_redhat:
+    :default_product_version_to_api_version_regex_hash:
```

Comment:
I think this isn't the "default", rather the only way to set the mapping. I'd also remove the "regex" and "hash" parts from the name, as those are implementation details. So I suggest to name this `product_version_to_api_versions`.